### DISCUSSION TITLE
Regression Recap Notes

### DISCUSSION THREAD

**Discussion by jmgirard** [affcomlab/learn-glms#4] [Ideas]
This is a place to store notes and discuss about the chapter on regression recap

**Comment by ammsimmons** [comment]
- install -> install.packages (link to helpful documents) 
-
- callout for why each package is necessary (one sentence each,
- why we use tidyverse for easy of analysis, easystats make tables output the models, ggeffects to plot the models)
- 
- call out: read_csv comes from tidyverse
- 
- explain the formula as used in the syntax of lm().
- separate MR into three models, two continous var, one dummy coded, and one with continous and dummy code.
-- categorial/dummy coding : means and contrasts
-- with continous and dummy coded: "we controlled"  

- before scatterplot, put histogram for both 
- ggplot also comes from tidyverse  (use as a footnote)

- tel them what model_paramters() is 
- explain the 95% confidence interval 
- one sd in yearas is 0.5 increase  sd in salary.
- explain using pipes (in introduction).

**Comment by ammsimmons** [comment]
New notes:

- for packages, use the website not the pdfs. 
-  link to how to use ggplot(), read this chapter for R for Data Science, or the ggplot2() book. 
- move lm() syntax callout before the fititng code block.
-  put 1 + predictor for the lm() syntax block 
- uncomment out the standardized_paramters(fit) 
- reread for typos. 
- adjusted R^2 typo. 
- remove "inference" parens
- mytheme invisible code chunk, and add "+mytheme" to every plot, and state in Intro page. 
- what does it mean to control for multple predictors, and why you wanna do that? (one sentence. For example:... "unique contribution of each predictor")... repeat what I had a little further down. 
- remove "and" in parameter table and interpretation
-  use the term "slopes", but in parens say (beta1 and beta2). 
- typo, subscripting the X2 and X2 (section 4.1)
- Why would we prefer zero order vs partial effects (decision you need to make in conjunction with your research question, they answer different questions etc). 
- "the unique effect when controlling for other predictors explicitly in the model and can't say anything when controlling for variables not in the model".
- "As a reminder, the $ extracts one of the columns in the dataframe" 4.3
- print out yearspubs (2.2)
-   move variable table to after loading it and printing yearspub (2.2). 
- equation for 4.2 categorical, C(i) needs to be subscripted
- 4.2.1, explicltly put (by = "sex)
- 4.2: "Note that if you have two more categories, you will use the exact same approach but have more than one slope". 
- 4.3.1: specifically, these means for males and females "averaged over" not "across". 
-4.4: beta symbol is not formatted properly. 
- 4.4: moderation is an umnbella term, and interaction is one type. SO JUST USE THE TERM "INTERACTION"... and say (which is a form of moderation). Also, say interaction terms is "multiplying the predictors together". 
- 4.4: Use the long form of interaction syntax. then callout for asterisk. 
- use print_md for 4.4 output of marginal effects. 
- 4.4: callout, the interaction was not significnt, but one of the simple slopes was and the other wasnt. 
- plots for assumptions: height 7, width 5 ... fig-dash ?

**Comment by ammsimmons** [comment]
**Still need to change the plot of assumptions with fig-dash??!**
